presenters/keyword_test.go

```diff
@@ -21,3 +24,25 @@ func TestFormattedCreatedAtWithValidTime(t *testing.T) {
 
 	assert.Equal(t, "December 16, 2020", result)
 }
+
+func TestKeywordResultWithValidKeywordModels(t *testing.T) {
+	nonAdwordLinks, _ := json.Marshal([]string{"non-ad-link1", "non-ad-link2"})
+	topPositionAdwordLinks, _ := json.Marshal([]string{"top-ad-link1", "top-ad-link2"})
+
+	keyword := models.Keyword{
+		Keyword:                 faker.Name(),
+		Status:                  models.Pending,
+		LinksCount:              10,
+		NonAdwordLinks:          nonAdwordLinks,
+		TopPositionAdwordsCount: 10,
+		TopPositionAdwordLinks:  topPositionAdwordLinks,
+		TotalAdwordsCount:       10,
+		HtmlCode:                "test-html",
+	}
+
+	keywordPresenter := KeywordPresenter{Keyword: keyword}
+	result := keywordPresenter.KeywordLinks()
+
```

Comment:
To be more complete, this test should also test every single attribute:

```
assert.Equal(t, "test-keyword", keywordPresenter.Keyword.Keyword)
assert.Equal(t, "pending", keywordPresenter.Keyword.Status)
#...
```

The fact the keyword model can be accessed through the KeywordPresenter class is a feature.

That's why in the setup step, here using `faker.Name()` is not the right decision to make assertions:

```
keyword := models.Keyword{
  Keyword: "test-keyword",
```

Reply:
Updated in 9443497a28ae6a66460d0b21de9655626d82df83